.travis.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         - cargo test --frozen
 
     - language: go
-      go: 1.9
+      go: 1.10
```

Comment:
related to my earlier comment re: `input-digest`, i think we can ensure this doesn't happen again by modifying the `install` section with:

```bash
bin/dep ensure -vendor-only
bin/dep status
```

Reply:
also, consider adding `bin/dep status` to Dockerfile-go-deps to catch the same issue during development.